ee/clickhouse/models/event.py

```diff
@@ -202,3 +201,69 @@ def determine_event_conditions(
             result += "AND event = %(event)s"
             params.update({"event": v})
     return result, params
+
+
+def get_event_count_for_team_and_period(
+    team_id: Union[str, int], begin: timezone.datetime, end: timezone.datetime
+) -> int:
+    result = sync_execute(
+        """
+        SELECT count(1) as count
+        FROM events
+        WHERE team_id = %(team_id)s
+        AND timestamp > %(begin)s
+        AND timestamp < %(end)s
```

Comment:
this doesn't match what we do for postgres, which is <= and >= based on what I assume `timestamp__gte` means (here it's < & >). It also doesn't match the queries below.